src/server/routes/legacyRoutes.ts

```diff
@@ -161,25 +163,20 @@ export const registerCurrenciesV1Route: Serverlet<
  * Response body: { '24': number, '1': number }
  */
 export const fetchCurrencyV1Route: Serverlet<ApiRequest> = async request => {
-  const { log, path, query } = request
+  const { query } = request
 
   const checkedQuery = checkPayload(asUserIdQuery, query)
   if (checkedQuery.error != null) return checkedQuery.error
-  const { userId } = checkedQuery.clean
+  // const { userId } = checkedQuery.clean
 
-  const match = path.match(/notifications\/([0-9A-Za-z]+)\/?$/)
-  const currencyCode = match != null ? match[1] : ''
+  // const match = path.match(/notifications\/([0-9A-Za-z]+)\/?$/)
+  // const currencyCode = match != null ? match[1] : ''
```

Comment:
I left code like this commented out, so we sill have a reference for what the payload looks like, even if we ignore it.